Gemfile

```diff
@@ -1,23 +1,27 @@
 source 'https://rubygems.org'
 
 group :lint do
-  gem 'foodcritic', '~> 3.0'
-  gem 'rubocop', '~> 0.18'
-  gem 'rainbow', '< 2.0'
+  gem 'foodcritic', '~> 5.0'
```

Comment:
Just noticed that one.  I see that Doug wants to wait until we get a chef-dk release out with Foodcritic 6.0, which will happen in the next few weeks.  For now this can stay at 5.0